### PR TITLE
Update to latest core api assembly

### PR DESCRIPTION
**Pull request by szilvaa**

### CODE REVIEW

File: tests/e2e/TestAppBundlesApi.cs [hunk 1]
```diff
@@ -71,5 +71,30 @@ public async void AppBundles_GetAll()
...
+        [Order(Weight = 1.4)]
+        public async void AppBundles_GetAllVersion()
+        {
+            using (Fixture.StartTestScope())
+            {
+                var list = await Fixture.DesignAutomationClient.GetAllItems(async page => await Fixture.DesignAutomationClient.GetAppBundleVersionsAsync(this.app.Id, page));
+                Assert.Contains(1, list);
+                Assert.Contains(2, list);
+                Assert.Distinct(list);
+            }
```
Comment (zhuliice) [inline tests/e2e/TestAppBundlesApi.cs:97]: I guess we are adding the missed tests before?

File: tests/e2e/appsettings.user.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "Forge": {
-    "ClientId": "<your client id>",
-    "ClientSecret": "<your client secret>"
-  }
+    "Forge": {
+        "ClientId": "CokbOqRvp41DVunD7lBO2ciiwJpltTAf",
+        "ClientSecret": "68d7Go7R9QtGDR9Z"
+    }
```
Comment (zhuliice) [inline tests/e2e/appsettings.user.json:5]: should we remove these credentials?

**Review by zhuliice — COMMENTED** [review]
I am wondering why this PR updates so many files? Do they failed the tests?

File: tests/e2e/appsettings.user.json (hunk 2 above)
Comment (szilvaa) [reply tests/e2e/appsettings.user.json]: Yes. I didn't mean to add these. Rotated the secret...

File: tests/e2e/TestAppBundlesApi.cs (hunk 1 above)
Comment (szilvaa) [reply tests/e2e/TestAppBundlesApi.cs:97]: Yes, some more tests..

**Comment by szilvaa** [comment]
> I am wondering why this PR updates so many files? Do they failed the tests?

Because these files are generated and the new code generates slightly different files.

**Review by zhuliice — APPROVED** [review]

**Comment by ricaun** [comment]
I wonder if this PR is related to the oauth version 2?
I noticed in the nuget has a new version, but in Github the release section was not update. Usually is a good ideia to release as well in the Github to match the version in the nuget, with a description in the release explain what change.

**Pull request merged**